spec/game_type_options_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "game_type_options"
+
+RSpec.describe GameTypeOptions do
+  HVH_OPTION = "Human Vs Human (1)"
+  HVB_OPTION = "Human Vs Easy AI (2)"
+  BVH_OPTION = "Easy AI Vs Human (3)"
+
+  it "gets game type options for display" do
+    expect(GameTypeOptions.for_display).to eq("#{HVH_OPTION} ; #{HVB_OPTION} ; #{BVH_OPTION}")
+  end
+
+  [1, "1", 2, "2", GameTypeOptions::HVH, GameTypeOptions::HVB, GameTypeOptions::BVH].map do |type| 
```

Comment:
What's the value of have multiple tests testing the same input? HVH == "1", so those two tests created here are exactly the same.